addons/web_tour/static/src/scss/tip.scss

```diff
@@ -114,11 +140,13 @@
 
     &::after {
         content: "";
-        @include o-position-absolute(-$o-tip-border-width, -$o-tip-border-width, -$o-tip-border-width, -$o-tip-border-width);
-        border-color: $o-tip-color;
-        border-radius: 50%;
-        transition: margin $o-tip-size-duration-out ease $o-tip-size-delay-out, border-radius $o-tip-duration-out linear 0s;
-        background: radial-gradient(lighten($o-tip-color, 7%), $o-tip-color);
+        :not(.o_animated) {
```

Comment:
That css is just not valid so it cannot be ready for review... an "::after" element cannot contain anything.